client/cmdhfmf.c

```diff
@@ -1530,6 +1557,423 @@ static int CmdHF14AMfNestedHard(const char *Cmd) {
+                            PrintAndLogEx(SUCCESS, "[REUSED  KEYS] Valid KEY FOUND: sector:%3d key type:%c key: " _YELLOW_("0x%02x%02x%02x%02x%02x%02x"), 
+                                current_sector_i,
+                                current_key_type_i ? 'B' : 'A',
+                                tmpKey[0], tmpKey[1], tmpKey[2], tmpKey[3], tmpKey[4], tmpKey[5]);
+                            foundkey = bytes_to_num(tmpKey, 6);
+                            foundKeysReuse++;
+                            break;
+                        }
+                    }
+                }
+            }
+            // Try the default keys
+            if (foundkey == 0) {
+                for (default_keys_i=0; default_keys_i < keycnt; default_keys_i++) {
+                    // Iterate over the keys
+                    for (i=0; i<6; i++) {
+                        tmpKey[i] = keyBlock[i + (6*default_keys_i)];
+                    }
+
+                    if (mfCheckKeys(current_sector_i*4, current_key_type_i, true, 1, tmpKey, &key64) == PM3_SUCCESS) {
+                        PrintAndLogEx(SUCCESS, "[Dictio. KEYS] Valid KEY FOUND: sector:%3d key type:%c key: " _YELLOW_("0x%02x%02x%02x%02x%02x%02x"), 
+                            current_sector_i,
+                            current_key_type_i ? 'B' : 'A',
+                            tmpKey[0], tmpKey[1], tmpKey[2], tmpKey[3], tmpKey[4], tmpKey[5]);
+                        foundkey = bytes_to_num(tmpKey, 6);
+                        foundKeysDictionary++;
+                        break;
+                    }
+                }
+            }
+            // Bruteforce with hardnested
+            if (foundkey == 0) {
+                PrintAndLogEx(SUCCESS, "[ BRUTEFORCE ] sector no:%3d, target key type:%c, Slow: %s, Tests: %d ",
+                    current_sector_i,
+                    current_key_type_i ? 'B' : 'A',
+                    slow ? "Yes" : "No",
+                    tests);
+
+                isOK = mfnestedhard(blockNo, keyType, key, current_sector_i*4, current_key_type_i, NULL, nonce_file_read, nonce_file_write, slow, tests, &foundkey, NULL);
+                
+                DropField();
+                if (isOK) {
+                    switch (isOK) {
+                        case 1 :
+                            PrintAndLogEx(ERR, "Error: No response from Proxmark3.\n");
+                            break;
+                        case 2 :
+                            PrintAndLogEx(NORMAL, "Button pressed. Aborted.\n");
+                            break;
+                        default :
+                            break;
+                    }
+                    free(e_sector);
+                    return 2;
+                }
+
+                num_to_bytes(foundkey, 6, tmpKey);
+                PrintAndLogEx(SUCCESS, "[CRACKED  KEY]  Valid KEY FOUND: sector:%3d key type:%c key: " _YELLOW_("0x%02x%02x%02x%02x%02x%02x"), 
```

Comment:
we are trying to avoid words like "CRACKED".   I suggest "FOUND"  instead. 

[FOUND] Valid key:

Reply:
DONE